src/EnergyPlus/DElightManagerF.cc

```diff
@@ -252,8 +252,8 @@ namespace DElightManagerF {
                     ShowWarningError(state, format("Maximum of 100 Reference Points exceeded for daylighting zone using DElight ={}", znDayl.Name));
                     ShowWarningError(state, "  Only first 100 Reference Points included in DElight analysis");
                 }
-                znDayl.DaylRefPtAbsCoord.allocate(3, znDayl.TotalDaylRefPoints);
-                znDayl.DaylRefPtAbsCoord = 0.0;
+                znDayl.DaylRefPtAbsCoord.allocate(znDayl.TotalDaylRefPoints);
```

Comment:
This was really a 1D array of points (i.e., `Vector3<Real64>`) masquerading as an `Array2D`.

Reply:
Nice.